plugins/package-managers/node/src/main/kotlin/utils/NpmSupport.kt

```diff
@@ -247,8 +247,10 @@ internal fun parsePackage(
     return module
 }
 
+private val logger = loggerOf(MethodHandles.lookup().lookupClass())
```

Comment:
I'd move this to the top to make more clear that all function could make use of it.

Reply:
Agree.